bin/reth/src/node/mod.rs

```diff
@@ -183,9 +190,32 @@ impl Command {
             info!(target: "reth::cli", "Continuous sync mode enabled");
         }
 
-        // TODO: This will be fixed with the sync controller (https://github.com/paradigmxyz/reth/pull/1662)
-        let (tx, _rx) = watch::channel(ForkchoiceState::default());
-        let engine_api_handle = self.init_engine_api(Arc::clone(&db), tx, &ctx.task_executor);
+        let (consensus_engine_tx, consensus_engine_rx) = unbounded_channel();
+
+        let _tip_rx = match self.tip {
+            Some(tip) => {
+                let (tx, rx) = oneshot::channel();
```

Comment:
I recommend we name the tx,rx vars properly because might be confusing since we have multiple